CourseKit/Source/UI/ViewControllers/LiveStreamContentViewController.swift

```diff
@@ -38,50 +39,75 @@ class LiveStreamContentViewController: BaseUIViewController {
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         stopReloadingContent()
+        player?.pause()
     }
     
-    func setupPlayerView(){
-        playerViewController = VideoPlayerViewController(hlsURL: content.liveStream!.streamURL, drmLicenseURL: nil)
-        
-        addChild(playerViewController!)
-        playerContainer.addSubview(playerViewController!.view)
-        playerViewController!.view.frame = playerContainer.bounds
-        playerViewController.playerView.isLive = true
+    func setupPlayerView() {
+        initializePlayer()
+        configurePlayerViewController()
+        configurePlayerView()
+        player?.play()
     }
```

Comment:
Please extract the player integration into a separate view controller in a separate PR. This way, both live and video content can use the same implementation, reducing code duplication. Additionally, if we need to modify the player configuration in the future, we’ll only need to update it in one place.